docs/user-guide/spark-user-guide.md

```diff
@@ -0,0 +1,159 @@
+
+There are potentially many input files needed for the CQL evaluation each representing a different flat data table. The physical location of these files is mapped to a logical data type name using one or more `-i` arguments to the Spark CQL Evaluator application. The `-i` arguments are expressed as datatype=uri pairs. For example, Patient data stored in an S3 endpoint would be expressed as `-i Patient=s3a://my-bucket/path/to/data`. The data type name here is important. This key is used downstream in the `context-definitions.json` file and also must correspond to a data type defined in the CQL modelinfo file. 
+
+The data being read can be in any format that Spark understands, but all data is assumed to be provided in a single, consistent format. The default format for Spark is parquet. Users can adjust the input format by setting the `spark.sql.datasources.default` configuration property when invoking the Spark application or by using the `--input-format` option to the application. The sample Dockerfile that is provided has support for all the basic formats supported natively by Spark and also Deltalake format. Users that want additional formats, such as Apache Iceberg, can use the `--packages` option to spark submit or extend the Docker image to add the additional needed JARs.
+
+Output is configured almost identically to input except it uses `-o` options and the key in the key=value pairs is the context name of a context aggregation defined in the `context-definitions.json` file. More on that later. Output format is, again, available in any supported spark format as specified by the `--output-format` program option, whatever is configured in the `spark.sql.sources.default` Spark configuration, parquet if no other configuration is specified.
```

Comment:
Output format is, again, available in any supported spark format as specified by the `--output-format` program option, whatever is configured in the `spark.sql.sources.default` Spark configuration, parquet if no other configuration is specified.

Should this read as: "Output format is, again, available in any supported spark format as specified by the `--output-format` program option or whatever is configured in the `spark.sql.sources.default` Spark Configuration. The output format is parquet if no other configuration is specified. ?